scripts/Run_uncrustify_for_sources.bat

```diff
@@ -0,0 +1,91 @@
+@echo off
+setlocal
+
+rem
+rem 14.12.2016
+rem
+rem Checks if all source files conform
+rem to the uncrustify configuration
+rem The check is performed with the uncrustify
+rem program that is available in the path.
+rem This might be an older version than
+rem a newly build one.
+rem 
+rem Call this batch script from the top level
+rem directory of the uncrustify project.
+
+set SRC_DIR=src
+set OUT_DIR=results
+set CFG_FILE=forUncrustifySources.cfg
+set DIFF_FILE=lastdiff.txt
+set "PWD=%cd%"
+
+rem Check if uncrustify is available in path
+set prog=uncrustify.exe
+for %%i in ("%path:;=";"%") do (
+    if exist %%~i\%prog% (
+		set found=%%i
```

Comment:
Wow, I looked it up, bat scripts actually do not have break statements. A solution would be to use goto but apparently that could, in some way, cause errors too as MS did to intend to let loops end prematurely.